### PR TITLE
Test step wait for stack analysis completion

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -139,6 +139,37 @@ def finish_analysis_for_component(context, ecosystem, component, version):
         raise Exception('Timeout waiting for the component analysis results')
 
 
+@when("I wait for stack analysis to finish")
+@when("I wait for stack analysis version {version} to finish")
+def wait_for_stack_analysis_completion(context, version=1):
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: `version="1"`

Comment (tisnik) [inline integration-tests/features/steps/common.py]: Nice catch, thans

**Comment by tisnik** [comment]
WDYT @fridex? 👍 ? :)

File: integration-tests/features/steps/common.py [hunk 2]
```diff
@@ -139,6 +139,37 @@ def finish_analysis_for_component(context, ecosystem, component, version):
...
+
+    Current API implementation returns just two HTTP codes:
+    200 OK : analysis is already finished
+    202 Accepted: analysis is started or is in progress (or other state!)
+    """
+
+    timeout = 600      # in seconds
+    sleep_amount = 10  # we don't have to overload the API with too many calls
+
+    id = context.response.json().get("id")
```
Comment (fridex) [inline integration-tests/features/steps/common.py]: Python's `id` is shadowed.

**Pull request merged**